### DISCUSSION TITLE
global env option

### DISCUSSION THREAD

**Discussion by jcook-uptycs** [evilmartians/lefthook#421] [Ideas]
I'm looking at how to set an env var more globally. I know this can be done via an rc file. But the rc file can't be added to the repo. 

Having a global `env` option would be very helpful. 

Or having an option to also use an rc file from the repo. (this would require allowing a relative path.)

Thanks

**Comment by mrexox** [comment]
You can set an ENV var for the specific command or script via [`env`](https://github.com/evilmartians/lefthook/blob/master/docs/configuration.md#env) option. For more convenience adding a global `env` can also work. Do you think it would be convenient?

**Reply by jcook-uptycs** [reply]
Using and global ENV on the shell side could work, but that becomes a pre-requirement every dev must update. Same with the existing rc file option.

Having it managed in the repos would be very helpful.

```yaml
env:
  REPORT_DIR: ./tmp/reports
  GIT_PAGER: cat
```
